### ISSUE TITLE
USER STORY: Predict Task Success notebook - ML pipeline with all data

### ISSUE THREAD

**Issue by Markus-Hefner** [Markus-Hefner/Portfolio-Project-5#12]
As a **client** I can **read through the notebook** so that **I can learn how the model performed with all variables as well as what the most important variables were**. 

**Acceptance criteria** 

AC1 - A  model to predict the task success of a developer given all variables
AC2 - An anaylsis and statement of what the most important variables were

**Tasks**

- [x] Create notebook
- [x] Create pipeline for data cleaning and feature engineering
- [x] Split train and test set
- [x] Handle target imbalance
- [x] Find the most suitable algorithm
- [x] Find best hyperparameter configuration (preliminary)
- [x] Assess feature importance
- [x] Evaluate pipeline on train and test set
- [x] Make sure that the model fulfills the business requirements measured by the metrics stated in the README file
